static/js/xmleditor/jquery.xmleditor.js

```diff
@@ -905,6 +905,18 @@ $.widget( "xml.xmlEditor", {
 		}
 		// convert XML DOM to string
 		var xmlString = this.xml2Str(this.xmlState.xml);
+
+		if (/mods:title.*?\/>/.test(xmlString)) {
+			if (window.confirm("The current MODS document has no title. " +
+				"This will result in no title displaying for the record. Would you like to save anyway?")) {
```

Comment:
This statement probably isn't accurate anymore. Maybe we can remove this warning, assuming the indexing part works correctly?